client/src/app/(modules)/network/(main)/(details)/suggest-button.tsx

```diff
@@ -0,0 +1,37 @@
+'use client';
+
+import { Pencil } from 'lucide-react';
+
+import { useGetNetworkSuggestion } from '@/types/generated/network-suggestion';
+
+import { Button } from '@/components/ui/button';
+
+const SuggestButton = ({
+  data,
+  id,
+  label,
+}: {
+  data: { name: string };
+  id: number;
+  label: 'project' | 'organisation';
+}) => {
+  const networkEmails = useGetNetworkSuggestion();
+  const emails = networkEmails.data?.data?.attributes?.email_recipients as string[] | undefined;
+  if (!emails || typeof emails === 'undefined') return null;
```

Comment:
I think this condition is redundant. If `emails` in `undefined` or `null`, then `!emails` will be truthy.
```suggestion
  if (!emails) return null;
```